core/tests/protractor_utils/forms.js

```diff
@@ -110,6 +111,8 @@ var GraphEditor = function(graphInputContainer) {
         // dict's edges.
```

Comment:
Above, you can replace `expect(await vertexElement(i).isDisplayed()).toBe(true);` with a `waitFor.visibilityOf()`. That will also cause the test to fail if the element isn't present, but it'll also wait for the element to load.

Reply:
updated!